OTKit/otkit-typography-desktop/token.yml

```diff
@@ -1,115 +1,143 @@
     type: "raw"
-    value: "{!font-weight-bold}"
-  heading-large-font-weight-alternate:
+    value: "normal"
+  xsmall-regular-line-height:
+    type: "size"
+    value: "16px"
+
+
+  # Design decisions for font group small-regular.
+  #
+  # =============================================
+  small-regular-font-size:
+    type: "size"
+    value: "14px"
+  small-regular-font-weight:
     type: "raw"
-    value: "{!font-weight-normal}"
-  heading-large-line-height:
+    value: "normal"
+  small-regular-line-height:
     type: "size"
-    value: "{!line-height-large}"
+    value: "20px"
+
 
-  heading-medium-font-size:
+  # Design decisions for font group small-medium.
+  #
+  # =============================================
+  small-medium-font-size:
     type: "size"
-    value: "{!font-size-medium}"
-  heading-medium-font-weight:
+    value: "14px"
+  small-medium-font-weight:
     type: "raw"
-    value: "{!font-weight-medium}"
-  heading-medium-line-height:
+    value: "500"
+  small-medium-line-height:
     type: "size"
-    value: "{!line-height-medium}"
+    value: "20px"
 
-  subheading-small-font-size:
+
+  # Design decisions for font group medium-bold.
+  #
+  # =============================================
+  medium-bold-font-size:
     type: "size"
-    value: "{!font-size-small}"
-  subheading-small-font-weight:
+    value: "24px"
+  medium-bold-font-weight:
     type: "raw"
-    value: "{!font-weight-medium}"
-  subheading-small-line-height:
+    value: "bold"
+  medium-bold-line-height:
     type: "size"
-    value: "{!line-height-small}"
+    value: "32px"
 
-  bodytext-small-font-size:
+
+  # Design decisions for font group large-bold.
+  #
+  # =============================================
+  large-bold-font-size:
     type: "size"
-    value: "{!font-size-small}"
-  bodytext-small-font-weight:
+    value: "24px"
+  large-bold-font-weight:
     type: "raw"
-    value: "{!font-weight-normal}"
-  bodytext-small-line-height:
+    value: "bold"
+  large-bold-line-height:
     type: "size"
-    value: "{!line-height-small}"
+    value: "40px"
+
 
-  subtext-xsmall-font-size:
+  # Design decisions for font group xlarge-bold.
+  #
+  # =============================================
+  xlarge-bold-font-size:
     type: "size"
-    value: "{!font-size-xsmall}"
-  subtext-xsmall-font-weight:
+    value: "32px"
+  xlarge-bold-font-weight:
     type: "raw"
-    value: "{!font-weight-medium}"
-  subtext-xsmall-font-weight-alternate:
+    value: "bold"
+  xlarge-bold-line-height:
+    type: "size"
+    value: "40px"
+
+
+  # Design decisions for font group xxlarge-bold.
+  #
+  # =============================================
+  xxlarge-bold-font-size:
+    type: "size"
+    value: "48px"
+  xxlarge-bold-font-weight:
     type: "raw"
-    value: "{!font-weight-normal}"
-  subtext-xsmall-line-height:
+    value: "bold"
+  xxlarge-bold-line-height:
     type: "size"
-    value: "{!line-height-xsmall}"
+    value: "56px"
+
 
 imports:
```

Comment:
shouldn't we remove the dependency from the aliases?

Reply:
@matteofigus yes, I can remove it 😆